src/cmd/node/rpc/gas_to_weight.rs

```diff
@@ -0,0 +1,60 @@
+use anyhow::Result;
+use clap::Parser;
+use jsonrpsee::core::client::ClientT;
+use jsonrpsee::http_client::HttpClientBuilder;
+use jsonrpsee::rpc_params;
+use serde::Deserialize;
+use std::fmt;
+
+/// Estimate gas for publishing modules.
+#[derive(Parser, Debug)]
+#[clap(about = "Convert gas to weight")]
+pub struct GasToWeight {
+    #[clap(short, long, help = "Gas should be an u64 value")]
+    gas: u64,
+}
+
+#[derive(Debug, Deserialize)]
+#[allow(unused)]
+struct Weight {
```

Comment:
As I understand that's defined here to avoid `frame_support` dependency.
We can think, however, at some point in the future, to add such deps if more things from Substrate will be needed.

Reply:
I agree - but I'm unsure what strategy to apply here. We can think about it more later.
It's sort of related to the answer [above](https://github.com/eigerco/smove/pull/11#discussion_r1432844040).